mentors/scala/b.json

```
@@ -1 +1,10 @@
-[]
+[
+{
+  "github_username": "leobenkel",
+  "name": "Leo Benkel",
+  "link_text": "leobenkel.com",
+  "link_url": "http://www.leobenkel.com/",
+  "avatar_url": null,
+  "bio": "I am an experienced developer and research scientist that is passionate about Sciences and Technologies. As a core team member of a successful startup, that was acquired by a multi billion dollar company, I have the experience of building products from scratch, creating value with customers, and integrating my efforts into giant Enterprise systems with different architectures. My personal love in the computer sciences are Artificial Intelligence and Multi Agent Systems. I enjoy designing and architecting libraries as well as software products, because of the interesting challenges they present. I love working with teams that want to create the 'possible' out of the 'improbable' while leveraging my creative mind and prototyping/ research skills. Simply, I love exploring where no-one has ever been before."
+}
```

```suggestion
  }
```

Also indent 2 spaces.